twitterCSS.css

```diff
@@ -0,0 +1,115 @@
+/* @media screen and (500px){
```

Comment:
Overall good work. There are few feedbacks with the page:
1. Good use of font that is similar to what Twitter page has. 
2. Good centering of elements on some elements although some elements could use further centering. 
3. That bird background is good. It is centered in the middle of the page which is what Twitter page has.
4. Good use of icons for blue side although texts could be bigger.
5. Little differences in font sizes and sizes of buttons.
6. Good use of favicon 
5. Page does not respond well when browser is resized which can be mitigated with use of flex. 

Other than above feedbacks, page looks fine and html elements are well organized with divs. Keep trying new CSS styles and you will create cool looking pages in no time.